integration-tests/src/test/java/oracle/kubernetes/operator/JrfInOperator.java

```diff
@@ -0,0 +1,179 @@
+
+  private static String operator1File = "jrfoperator1.yaml";
+
+  // file used to customize domain properties for domain, PV and LB inputs yaml
+  private static String jrfdomainonpvwlstFile = "jrfdomainonpvwlst.yaml";
+
+  // property file for oracle db information
+  private static String dbPropsFile = "oracledb.properties";
+
+  // property file used to configure constants for integration tests
+  private static String appPropsFile = "OperatorIT.properties";
+
+  private static Operator operator1;
+
+  private static boolean QUICKTEST;
+  private static boolean SMOKETEST;
+  private static boolean JENKINS;
+
+  // Set QUICKTEST env var to true to run a small subset of tests.
+  // Set SMOKETEST env var to true to run an even smaller subset
+  static {
+    QUICKTEST =
+        System.getenv("QUICKTEST") != null && System.getenv("QUICKTEST").equalsIgnoreCase("true");
+    SMOKETEST =
+        System.getenv("SMOKETEST") != null && System.getenv("SMOKETEST").equalsIgnoreCase("true");
+    if (SMOKETEST) QUICKTEST = true;
+    if (System.getenv("JENKINS") != null) {
+      JENKINS = new Boolean(System.getenv("JENKINS")).booleanValue();
+    }
+  }
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes. It also creates Oracle DB pod which used for
+   * RCU.
+   *
+   * @throws Exception - if an error occurs when load property file or create DB pod
+   */
+  @BeforeClass
+  public static void staticPrepare() throws Exception {
+    // initialize test properties and create the directories
+    initialize(appPropsFile);
+
+    // create DB used for jrf domain
+    OracleDB db = TestUtils.createOracleDB(dbPropsFile);
+
+    // populate the jrf/create-domain-script.sh
+    // copy the integration-tests/src/test/resources/domain-home-on-pv/jrf to
+    // BaseTest.getResultDir()
+    TestUtils.exec(
+        "cp -rf "
+            + BaseTest.getProjectRoot()
+            + "/integration-tests/src/test/resources/domain-home-on-pv/jrf "
+            + BaseTest.getResultDir());
+    // replace the db connection string with true value
+    String dbConnectString =
+        db.getName()
+            + "."
+            + db.getNamespace()
+            + ".svc.cluster.local:"
+            + db.getPort()
+            + "/"
+            + db.getDBPdb()
+            + "."
+            + db.getDBDomain();
+    TestUtils.replaceStringInFile(
+        BaseTest.getResultDir() + "/jrf/create-domain-script.sh",
+        "%CONNECTION_STRING%",
+        dbConnectString);
+  }
+
+  /**
+   * This method will run once after all test methods are finished. It Releases k8s cluster lease,
+   * archives result, pv directories.
+   *
+   * @throws Exception - if any error occurs
+   */
+  @AfterClass
+  public static void staticUnPrepare() throws Exception {
```

Comment:
you may want to check the latest IT* classes in develop branch, this code is moved to BaseTest.